phase2/kubeadm/configure-vm-kubeadm-master.sh

```diff
@@ -43,4 +42,25 @@ kubeProxy:
 EOF
 fi
 
+# if exist $KUBEADM_FEATURE_GATES is an simple array with "key1 value1 key2 value2 .."
+# it should be added in the config like:
+# featureGates:
+#   key1: value1
+#   key2: value2
+#   etc ...
+featuresdata=($KUBEADM_FEATURE_GATES)
+featureslength=(${#featuresdata[*]})
```

Comment:
I am worried using `${` might not work for startup scripts. In openstack implementation we are using terraform templating which has special treatment for the patterns with `${`. Please refer https://www.terraform.io/docs/configuration/interpolation.html

Can we have some other alternative and avoid using `${` ?

Reply:
hum.. let me check what I can do based on this link.